### DISCUSSION TITLE
YouTuber Jeff Su mentioned that CMD + CTRL + brightness up/down can be used to change both display brightness

### DISCUSSION THREAD

**Discussion by Tech-Aesthetic** [MonitorControl/MonitorControl#1094] [Q&A]
Hi, like the title says, I watched this YouTuber's video and he explicitly mentioned at [03:20](https://youtu.be/-xXc7qeiC8I?t=199) that you can use the command control brightness keys keyboard shortcut to change your monitors' brightness. I know there are other Monitor Control settings preferences that let you sync brightness, but I'm wondering if this keyboard short cut is still there and how to get it to work, because for me that doesn't change both monitors' brightness. 

Thank you for your help.

**Comment by waydabber** [comment]
there is no shortcut to control all displays. There is one to control the internal display and an other to control all external displays, maybe the youtuber was referring to this.
